src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
+| Gold mining         |                 240                 |          92,000x           | [source](https://www.kitco.com/news/2021-05-17/Gold-s-energy-consumption-doubles-that-of-bitcoin-Galaxy-Digital.html)                             |
+| Gold mining         |                 130                 |          50,000x           | [source](https://ccaf.io/cbeci/index/comparisons)                                                                                                 |
+| Bitcoin             |                 130                 |          50,000x           | [source](https://digiconomist.net/bitcoin-energy-consumption)                                                                                     |
+| Bitcoin             |                 100                 |          38,000x           | [source](https://ccaf.io/cbeci/index/comparisons)                                                                                                 |
+| Youtube             |                 244                 |          94,000 x          | [source](https://thefactsource.com/how-much-electricity-does-youtube-use/)                                                                        |
+| Global data centers |                 200                 |          78,000x           | [source](https://www.iea.org/commentaries/data-centres-and-energy-from-global-headlines-to-local-headaches)                                       |
+| Netflix             |                0.45                 |            175x            | [source](https://s22.q4cdn.com/959853165/files/doc_downloads/2020/02/0220_Netflix_EnvironmentalSocialGovernanceReport_FINAL.pdf)                  |
+| Netflix             |                 94                  |          36,000x           | [source](https://theshiftproject.org/en/article/unsustainable-use-online-video/)                                                                  |
+| Paypal              |                0.26                 |            100x            | [source](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)                       |
+| Gaming in USA       |                 34                  |          13,000x           | [source](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential) |
+| PoW Ethereum        |                 78                  |          30,000x           | [source](https://digiconomist.net/ethereum-energy-consumption)                                                                                    |
+| PoS Ethereum        |               0.0026                |             1x             | [source](https://carbon-ratings.com/eth-report-2022)                                                                                              |
```

Comment:
> 1x

Should we just omit this for PoS Ethereum (i.e. dash it out)?

Reply:
I think it works as is... makes it clear this is the factor being compared against

Reply:
I like it too - it makes it clear what the metric means in the other rows